.appveyor.yml

```diff
@@ -99,6 +86,4 @@ build_script:
   - md build
   - cd build
   - cmake --version
-  - if NOT DEFINED PLATFORM cmake %AIRPCAP% %REMOTE% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DAirPcap_ROOT=c:\projects\libpcap\Win32\Airpcap_Devpack -G"%GENERATOR%" ..
```

Comment:
These are the lines that run CMake to configure the build, and cannot and must not be removed; otherwise, the build fails, as it's doing on AppVeyor.

Just remove `%AIRPCAP%` from the arguments to CMake.